addon/components/dropdown-action.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { scheduleOnce } from '@ember/runloop';
+import { tryInvoke } from '@ember/utils';
+export default class DropdownActionComponent extends Component {
```

Comment:
since all the classic ember component element customization properties are removed from the glimmer component class, we need to define a `dropdown-action.hbs` template file where those'll all live now, e.g.:

```hbs
<button
    class="dropdown-action dropdown-item"
    type="button"
    {{on "click" this.onClick}}
    ...attributes
>
    {{yield}}
</button>
```